Git.hub.Demo/Git.hub.Demo.csproj

```diff
@@ -23,6 +23,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
:bulb; A better way to do this is using a rule set file to actually set the severity of the original issue to Error instead of Warning.